addon/services/flexberry-logging.js

```diff
@@ -14,22 +14,45 @@ const { getOwner } = Ember;
  */
 export default Ember.Service.extend({
   /**
-    Store for transmit messages to server.
+   *   Current log level. Set this property to log appropriate interval of log types
+   *
+   *   @property {Integer} flexberryLogLevel
+   */
+  flexberryLogLevel: 0,
 
-    @property flexberryStore
-    @type DS.Store
-    @default null
+  /**
+   * Log level enumarator
+   *
+   * @property logLevelEnums
+   * @type Object
+   */
+  logLevelEnums: {
+    ERROR: 1, // Log only errors
+    WARN: 2,  // Log warnings and errors
+    LOG: 3, // Log logs, warnings and errors
+    INFO: 4,  // Log infos, logs, warnings and errors
+    DEBUG: 5,// Log debugs, infos, logs, warnings and errors
+    DEPRECATION: 6 // Log deprecations, debugs, infos, logs, warnings and errors
+  },
+  /**
+   * Inverted to logLevelEnums array
+   *
+   * @property enumsLoglevel
+   * @type Array
    */
-  flexberryStore: null,
+  enumsLoglevel: ['NONE'],
+  /**
+   store for transmit messages to the server
 
+   @property {Object} _flexberryStore
+   */
+  _flexberryStore: null,
   /**
-    Logger switcher on/off true if remote logging service works correctly.
+   Logger switcher on/off true if remote logging service works correctly
 
-    @property serverLogEnabled
-    @type Boolean
-    @default true
+   @property {Boolean} _serverLogEnabled
    */
-  serverLogEnabled: true,	//Remote logging service works correctly
+  _serverLogEnabled: true,	//Remote logging service works correctly
```

Comment:
"@property {Boolean} _serverLogEnabled" - it is not yuidoc.
it should be marked as private.
private variable has to be placed before public.